### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by aidenscott2016**

### CODE REVIEW

File: example/nixos-flake/luks-lvm/configuration.nix [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{ config, pkgs, ... }:
+
+{
+  # point root to the physical partition that your LUKS encrypted LVM volume is on
+  boot.initrd.luks.devices = {
```
Comment (Lassulus) [inline example/nixos-flake/luks-lvm/configuration.nix:5]: why is this needed? disko should do this for you

**Comment by Mic92** [comment]
Cherry picked cli changes in https://github.com/nix-community/disko/pull/139/commits/a9e0f9ba442931b87fd9f2eedc54d6dcedddf074

**Comment by Mic92** [comment]
I also added a quick start guide.

File: example/nixos-flake/luks-lvm/configuration.nix (hunk 1 above)
Comment (aidenscott2016) [reply example/nixos-flake/luks-lvm/configuration.nix:5]: thanks! I didn't know this

**Comment by aidenscott2016** [comment]
Completely forgot I started working on this!
